py3-libarchive-c.yaml

```
@@ -0,0 +1,40 @@
+package:
+  name: py3-libarchive-c
+  version: 5.0
+  epoch: 0
+  description: "Python interface to libarchive."
+  copyright:
+    - license: CC0-1.0
+
```

Since this is some FFI thing, you probably need to add a runtime dependency on libarchive:

```suggestion
  dependencies:
    runtime:
      - libarchive
```

---

Thanks!